test/threads.jl

```diff
@@ -274,8 +274,12 @@ let atomic_types = [Int8, Int16, Int32, Int64, Int128,
     # Temporarily omit 128-bit types on 32bit x86
     # 128-bit atomics do not exist on AArch32.
     # And we don't support them yet on power.
+    # LLVM 3.9 on windows is missing support as well.
+    # This might be fixed by switching to a mingw64 version
+    # that supports libatomic.
```

Comment:
Seems like a regression in functionality that we need external versions that we didn't before? Would rather leave this as a failure until we know what caused it on the LLVM side.